src/Text/Markdown/SlamDown/Parser/Inline.purs

```diff
@@ -427,12 +435,23 @@ parseTextBox isPlainText eta template =
       when (day > 31) $ P.fail "Incorrect day"
       pure { month, day, year }
 
-    parseTimeValue = do
+    parseTimeValue prec = do
       hours ← natural
       when (hours > 23) $ P.fail "Incorrect hours"
       PU.skipSpaces *> colon *> PU.skipSpaces
       minutes ← natural
       when (minutes > 59) $ P.fail "Incorrect minutes"
+      seconds ← case prec of
+        SD.Minutes -> do
```

Comment:
Maybe we could omit `fail` here, i.e set it to zero and pretend that user hasn't made mistake?